content/advanced/modules/prerequisites/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Prerequisites"
+weight: 310
+---
+
+### Module Development Tools
+
+Before proceeding to the next section, choose to install the following prerequisites on your machine:
+
+* install the [CloudFormation Command Line Interface (CLI)](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html).
+
+  Use `pip` to install the CloudFormation CLI:
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  pip install cloudformation-cli
```

Comment:
Should we recommend using a Python environment here? Or is that out of scope?

Reply:
I considered it out of scope and likely to open a "can of worms".